<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoSelectionPage1</class>
 <widget class="QWidget" name="VideoSelectionPage1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="path_text_label"/>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="browse_button">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 64, 130);
font: 10pt &quot;Rubik Light&quot;;
color: rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>Sfoglia</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="ready_button">
     <property name="styleSheet">
      <string notr="true">border-radius: 20px;
background-color: rgb(0, 64, 130);
font: 12pt &quot;Rubik Light&quot;;
color: rgb(255,255,255);</string>
     </property>
     <property name="text">
      <string>Pronto</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="title_label">
     <property name="styleSheet">
      <string notr="true">font: 25pt &quot;Rubik Light&quot;;
color: rgb(185,185,185);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Titolo</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="text_label">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Rubik Light&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>testo</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="error_text_label">
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Rubik Light&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../pages from main - qt designer/GUI/qtdesigner/logo_unimi.qrc"/>
 </resources>
 <connections/>
</ui>
